src/ixdat/plotters.py

```diff
@@ -0,0 +1,32 @@
+"""Classes for plotting measurement data"""
+
+from matplotlib import pyplot as plt
+
+
+class ValuePlotter:
+    """Default plotter. By default plots all of the VSeries vs time on a single axis"""
+
+    def __init__(self, measurement=None):
+        self.measurement = measurement
+
+    def plot(self, *args, **kwargs):
+        """Plot the the exporter's measurement via plotter.plot_measurement()"""
+        return self.plot_measurement(measurement=self.measurement, *args, **kwargs)
+
+    def plot_measurement(self, measurement, v_list=None, tspan=None):
+        """Plot a measurement.
+
+        Args:
+            measurement (Measurement): The measurement to export
+            v_list (list of str): The names of the data series to include. Defaults to
+                names of all VSeries in the measurement.
+            tspan (timespan): The timespan to include in the file, defaults to all of it
+        """
+        fig, ax = plt.subplots()
```

Comment:
I thought all plotting functions took ax as optional argument, that would be really great and should be in a consistent manner

Reply:
True! Forgot last time (would of course have remembered later :wink:). Fixed.
Now takes ax argument, None by default, only makes a new one if needed:
```
        if not ax:
            fig, ax = plt.subplots()
```